web/app/js/components/util/Utils.test.js

```diff
@@ -124,4 +125,18 @@ describe('Utils', () => {
       expect(toClassName('test/potato-e1af21-f3f3')).toEqual('test_potato_e_1_af_21_f_3_f_3');
     });
   });
+
+  describe('regexFilterString', () => {
+    it('converts string to a valid regex when filtering namespace selection options', () => {
```

Comment:
this method is used for other things besides filtering namespace selectors (e.g. in `BaseTable` to filter table data) - so I'd suggest we title it less specifically to Namespaces
```suggestion
  it('converts input string to a valid regex for filtering', () => {
```